# 电机基本参数
int32 device_id

# 电机控制模式
string control_mode

# 电机状态参数
string state

# 电机传感器参数
int64 sensor_position
int32 sensor_velocity
int16 sensor_temp
